src/registry.c

```diff
@@ -711,6 +711,7 @@ extract_text(xmlNode *node)
 static bool
 parse_config_item(struct rxkb_context *ctx,
                   xmlNode *parent,
+                  enum rxkb_popularity *popularity,
                   char **name,
                   char **description,
                   char **brief,
```

Comment:
not directly related to your patch but with all the args it would be better to have a 

```c
struct config_item {
   char *name;
   ...
}
```
and return that, together with a `config_item_free()` that moves the code into a single location. This will clean up the code a bit.

Reply:
Yep, I am all for cleaning up!